src/components/Basics.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import generateKey from '@tammiec/generatekey';
+
+import './Basics.scss';
+
+const Basics = ({ data, ...props }) => {
+
+  const profiles = data.profiles.map(profile => (<p key={generateKey()}>{profile.network}: <a href={profile.url}>@{profile.username}</a></p>));
```

Comment:
You can use an `useMemo` if you desire to avoid re-creating the profiles constants for each re-render unless this data it's dynamic at some point.